integration-tests/hibernate-orm-panache/src/main/java/io/quarkus/it/panache/DogDto2.java

```diff
@@ -3,7 +3,7 @@
 import io.quarkus.hibernate.orm.panache.common.NestedProjectedClass;
 import io.quarkus.runtime.annotations.RegisterForReflection;
 
-@RegisterForReflection(ignoreNested = false)
+@RegisterForReflection
```

Comment:
You will need to do the same in reverse as well, i.e., change `@RegisterForReflection` to `@RegisterForReflection(ignoreNested = true)` for registered classes that have nested classes. I expect there will be some.

Reply:
Not sure I find them all :)

Reply:
I performed a structural search in IntelliJ and it looks like you got them all. Note however that we need to do the same for cases like https://github.com/quarkusio/quarkus/blob/d92a2438cfce0908818e8cdb22579d4c4cf80ec0/devtools/cli/src/main/java/io/quarkus/cli/core/Reflections.java#L5-L42 where we register different classes than the annotated one (which is unfortunately is not possible to catch with structural search).

Reply:
Should we add the @RegisterForReflection in each class inside targets ? (of course just for io.quarkus classes)
Or just in the class declaring the targets ?

Reply:
No, just the class declaring the targets. If that doesn't work as expected I would consider it a bug :)